src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSBE/PartsParam.cs

```diff
@@ -782,17 +784,17 @@ public class UnkStruct1
                 /// <summary>
                 /// Unknown.
                 /// </summary>
-                public uint[] DisplayGroups { get; private set; }
+                public uint[] DisplayGroups { get; set; }
```

Comment:
with all the choices to private indexes for sanity, why are we making these setters public?
being an array, it's already an immutable reference to a correctly-size mutable array? maybe that's not how c# handles it idk

Reply:
This field is not backed up by any private fields so a private setter makes little sense
Main use case is XML serialization as it would fail on this